pkg/option/config.go

```diff
@@ -3354,8 +3371,23 @@ func InitConfig(programName, configName string) func() {
 			} else {
 				// replace deprecated fields with new fields
 				ReplaceDeprecatedFields(m)
-				err := MergeConfig(m)
-				if err != nil {
+				var confArgs []string
+				for key := range m {
+					value := fmt.Sprintf("%v", m[key])
+					// check whether the flag exists
+					if IsKnownFlag(key) {
+						if value != "" {
+							value = strings.Join([]string{"=", value}, "")
+							confArgs = append(confArgs, strings.Join([]string{"--", key, value}, ""))
```

Comment:
Go supports string concatenation with `+`, you don't need to use `strings.Join` and `append`. For example, these two lines are equivalent to:

```go
confArgs = append(confArgs, "--" + key "=" + value)
```

Reply:
Done